### PR TITLE
Attendee registration wording

### LINKED ISSUE #87: Provide flow dependent help text / information on registration and login pages
**Is your feature request related to a problem? Please describe.**
There are multiple ways how users of the website can reach the registration and login pages. It may be hard for the user to see the reason especially if the user gets interrupted between steps of the link flow.

**Describe the solution you'd like**
Add an optional URL parameter to the register and login URLs that triggers the display of a flow specific message.

**Describe alternatives you've considered**
Separate login and register pages for each flow would be an alternative but would lead to a lot of code duplication and future maintenance issues.

### PR DESCRIPTION
**Pull request by jandd**
fixes #87

### CODE REVIEW

File: devday/attendee/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -67,36 +67,37 @@ msgstr "Passwort ändern"
...
 msgid ""
-"<p class=\"text-info\">By registering as an attendee, I agree to be "
-"contacted by the Dev Day organizers about conference details.</p>"
+"By clicking the button below I express my intent to attend the event {event} "
+"and agree to be contacted by the Dev Day organizers about conference details."
 msgstr ""
-"<p class=\"text-info\">Durch meine Registrierung als Teilnehmer stimme ich "
-"zu, dass mich die Dev Day-Organisatoren über Details zur Konferenz "
-"kontaktieren dürfen.</p>"
+"Durch Klick auf den unten stehende Button bestätige ich meine Absicht an der "
```
Comment (snoopotic) [inline devday/attendee/locale/de/LC_MESSAGES/django.po]: stehendeN

File: devday/attendee/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -463,6 +464,35 @@ msgstr "Bitte fülle die Informationen für dein Badge aus."
...
+
+#: attendee/templates/attendee/attendee_registration.html:17
+#, python-format
+msgid ""
+"You need an account on this site to attend %(event_title)s. Please register "
+"an account using your email address."
+msgstr ""
+"Du benötigst ein Benutzerkonto auf dieser Seite, um an der Veranstaltung "
+"%(event_title)s teilzunehmen. Bitte registriere ein Benutzerkonto durch "
+"Angabe deiner E-Mailadresse."
```
Comment (snoopotic) [inline devday/attendee/locale/de/LC_MESSAGES/django.po]: E-Mail-Adresse

File: devday/attendee/locale/de/LC_MESSAGES/django.po (hunk 1 above)
Comment (jandd) [reply devday/attendee/locale/de/LC_MESSAGES/django.po]: fixed

File: devday/attendee/locale/de/LC_MESSAGES/django.po (hunk 2 above)
Comment (jandd) [reply devday/attendee/locale/de/LC_MESSAGES/django.po]: replaced throughout the code base

**Pull request merged**